### PR TITLE
MDEV-8579 Expand system variable documentation

### PR DESCRIPTION
**Pull request by iangilfillan**

### CODE REVIEW

File: sql/sys_vars.cc [hunk 1]
```diff
@@ -4703,13 +4790,17 @@ static Sys_var_charptr Sys_wsrep_cluster_address (
        ON_UPDATE(wsrep_cluster_address_update));
 
 static Sys_var_charptr Sys_wsrep_node_name (
-       "wsrep_node_name", "Node name",
+       "wsrep_node_name", "Name of this node. This name can be used in "
+       "wsrep_sst_donor as a preferred donor. Note that multiple nodes "
+       "in a cluster can have the same name. ",
```
Comment (svoj) [inline sql/sys_vars.cc]: Why space at the end?

File: sql/sys_vars.cc [hunk 2]
```diff
@@ -4069,51 +4135,69 @@ static Sys_var_charptr Sys_slow_log_path(
...
-       "have_openssl", "have_openssl",
+       "have_openssl", "Comparing have_openssl with have_ssl will indicate whether "
+       "YaSSL or openssl was used. If YaSSL, have_ssl will be YES, but have_openssl "
+       "will be NO.",
        READ_ONLY GLOBAL_VAR(have_openssl), NO_CMD_LINE);
 
 static Sys_var_have Sys_have_profiling(
-       "have_profiling", "have_profiling",
+       "have_profiling", "If statement profiling is available, will be set to YES, "
+       "otherwise will be set to NO. See SHOW PROFILES() and SHOW PROFILE().",
```
Comment (svoj) [inline sql/sys_vars.cc]: Hmm... why ```SHOW PROFILES()``` instead of ```SHOW PROFILES```?

File: sql/sys_vars.cc [hunk 3]
```diff
@@ -3686,13 +3745,16 @@ static bool update_profiling(sys_var *self, THD *thd, enum_var_type type)
 }
 
 static Sys_var_bit Sys_profiling(
-       "profiling", "profiling",
+       "profiling", "If set to 1 (0 is default), statement profiling will be "
+       "enabled. See SHOW PROFILES() and SHOW PROFILE().",
```
Comment (svoj) [inline sql/sys_vars.cc]: Hmm... why ```SHOW PROFILES()``` instead of ```SHOW PROFILES```?

**Comment by iangilfillan** [comment]
I have made the changes you suggested. I don't have a test environment to look at the failing tests, so if you can look at that when merging.

**Comment by svoj** [comment]
Thanks! I'll fix tests before merging.

**Comment by svoj** [comment]
Struggling with https://github.com/mariadb/server/commit/aec43216c834c2156e9aca7b7dc182fbdd1916de

Thus it is a bit delayed.

**Comment by svoj** [comment]
Merged to 10.1 with tests fixed.

**Pull request closed without merging**